src/index.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
```

Comment:
Please, don't remove the app component because it represents the application instance, and in this component could be executed the actions related to the application.